core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -106,16 +119,30 @@ THE SOFTWARE.
                       <a href="${%wiki.url}" target="_blank"><l:icon class="icon-help icon-sm" style="vertical-align:top"/></a>
                     </j:if>
                   </td>
-                  <td class="center pane">
+                  <td class="center pane uninstall">
                     <j:choose>
                       <j:when test="${p.isDeleted()}">
                         <p>${%Uninstallation pending}</p>
                       </j:when>
-                      <j:when test="${!p.isBundled()}">
```

Comment:
This is wrong -- Jenkins now offers to uninstall plugins that are bundled, and they only get reinstalled upon restart (for now). Files [JENKINS-31605](https://issues.jenkins-ci.org/browse/JENKINS-31605).